pkg/api/openshiftcluster.go

```diff
@@ -201,7 +201,6 @@ type InstallPhase int
 
 // InstallPhase constants
 const (
-	InstallPhaseDeployStorage InstallPhase = iota
-	InstallPhaseDeployResources
+	InstallPhaseDeploy InstallPhase = iota
```

Comment:
I dont know. I still reluctant on this. Install phase in my head should encapsulate only installer work, and we should have separate phase for "post install" things. And remove bootstrap is last. Im fine merging it, but I think this is not the solution to condense long running operations to even more long running operations.

Reply:
We have to minimise the number of overt install phases otherwise RP upgrade becomes extremely challenging.